pkg/trace/api/container_linux.go

```diff
@@ -0,0 +1,71 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+
+package api
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"syscall"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/log"
+	"github.com/DataDog/datadog-agent/pkg/util/containers/v2/metrics"
+)
+
+type ucredKey struct{}
+
+// cacheDuration determines how long a pid->container ID mapping is considered valid. This value is
```

Comment:
```suggestion
// cacheDuration determines how long a pid->container ID mapping is considered fresh. This value is
```

Reply:
I want to push back on this one a little bit. I'm OK using something other than "valid", but "fresh" confuses me a little. 

It's less clear to me what the behavior of a "fresh" vs "stale" cache value is. Do we still use the stale value, but try to update it? 

I would like to indicate that cache values get dropped after this period of time.